src/redux/auth/auth.sagas.js

```diff
@@ -21,7 +21,7 @@ function* handleAdminLoad({ payload }) {
   try {
     yield put(setAuthLoading(true));
     const admin = yield call(loginAdmin, payload);
-    setToLocalStorage('HORONDI_AUTH_TOKEN', admin.token);
+    yield call(setToLocalStorage,'HORONDI_AUTH_TOKEN', admin.token);
```

Comment:
There is no need to use `yield call` for access to the local storage because it is sync action from the js point of view

Reply:
fixed